addons/respawn/functions/fnc_showFriendlyFireMessage.sqf

```diff
@@ -1,25 +1,35 @@
+#include "script_component.hpp"
 /*
  * Author: commy2
  * Shows a message in system chat of who killed whom.
  *
  * Arguments:
- * 0: Unitn <OBJECT>
+ * 0: Unit <OBJECT>
  * 1: Killer <OBJECT>
  *
  * Return Value:
  * None
  *
  * Example:
- * [ACE_Player, killer] call ace_module_fnc_functionName
+ * [ACE_Player, killer] call ace_respawn_fnc_showFriendlyFireMessage
  *
  * Public: No
  */
-#include "script_component.hpp"
-
 params ["_unit", "_killer"];
 
+// fix, because ace medical cause _unit == _killer
+if (_unit == _killer) then {
+   _killer = _unit getVariable [QEGVAR(medical,lastDamageSource), _killer];
```

Comment:
This functions runs globally, but the medical var lastDamageSource is only set where unit is local.
Will probably want to determine this in fnc_handleDamage
ref https://github.com/acemod/ACEX/blob/master/addons/killtracker/XEH_postInit.sqf